code/iaas/model/src/test/java/io/cattle/platform/core/util/PortSpecTest.java

```diff
@@ -12,31 +12,31 @@ public void testValidFormats() {
         // format: ip:hostPort:containerPort | ip::containerPort | hostPort:containerPort | containerPort
         PortSpec spec = new PortSpec("80");
 
-        assertEquals(80, spec.getPrivatePort());
+        assertEquals(new Integer(80), spec.getPrivatePort());
```

Comment:
we should revert this change. Private port can never be null, and it should be primitive.